provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.json

```diff
@@ -23,6 +23,7 @@
     "SOME_MACHINE_ID": {
       "cluster_id": "my-cluster",
       "hostname": "myhost",
+      "name": "mynodename",
```

Comment:
Please also update create-template.sh script. In fact, this particular template is created by means of that script.